docs/BACKENDS.md

```
@@ -131,16 +157,221 @@ Suggestion: try using a non-portable backend first (e.g. win32 + underlying grap
+  - Use `io.AddFocusEvent()` to notify when application window gets focused/unfocused.
+- **Multi-viewport Support**
+  - Update monitor list if supported.
+  - Setup all required handlers in `ImGuiPlatformIO` to create/destroy/move/resize/title/focus/etc. windows.
+
+### Rendering: Implementing your RenderDrawData function
+
+Note: set `ImGuiBackendFlags_RendererHasVtxOffset` to signify your backend can handle rendering with a vertex offset (`ImDrawCmd::VtxOffset` field).
+Otherwise, rendering will be limited to 64K vertices per window, which may be limiting for advanced plot.
+As an alternative, you may also use `#define ImDrawIdx unsigned int` in your `imconfig.h` file to support 32-bit indices.
+
+```cpp
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)